packages/@okta/vuepress-site/docs/guides/set-up-event-hook/ongoing-delivery-of-events/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Ongoing Delivery of Events
+---
+
+After your event hook is verified, delivery of events to your endpoint can begin. There can be a delay, however, of up to a minute between successful verification and the start of event delivery.
```

Comment:
nit : Maybe `There can be a delay, however, of up to a minute between successful verification and the start of event delivery.` is more relevant after the next line which explains when an event hook is fired?
The 1 minute delay is related to `Every time an event of a type that the Event Hook is subscribed to occurs in your Okta org...` part.

Reply:
Have removed the statement. Need to follow up because something similar was stated in the Event Hooks conceptual overview page.

Reply:
Sorry for the confusion. Let me clarify - 
the statement is correct

I just meant maybe it should be moved down a little bit in terms of structure.
The way it is right now, it could also be interpret to mean `Once I verify hook, delivery takes up to a minute to start`. While it actually is - `Once I verify a hook, it takes up to a minute for Okta to know eligible events are to be delivered to this hook`. That is, an event fired immediately after verification may not be recognized by Okta as an event to deliver.

I guess its too subtle. I'll let you decide if the current wording / structure is fine. But we should definitely have it in.

Reply:
Thanks Amogh, that makes sense. I've updated to: "There can be a delay, however, of up to a minute between successful verification of an Event Hook and it becoming operational."